
> tribeminer@1.0.0 server
> tsx src/backend/server.ts

node:events:495
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3001
    at __node_internal_captureLargerStackTrace (node:internal/errors:496:5)
    at __node_internal_uvExceptionWithHostPort (node:internal/errors:593:12)
    at Server.setupListenHandle [as _listen2] (node:net:1811:16)
    at listenInCluster (node:net:1859:12)
    at Server.listen (node:net:1947:7)
    at <anonymous> (/home/oartes/projects/testing/TribeMiner by BitTribe/src/backend/server.ts:331:8)
    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:336:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:106:12)
Emitted 'error' event on WebSocketServer instance at:
    at Server.emit (node:events:517:28)
    at emitErrorNT (node:net:1838:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 3001
}

Node.js v18.19.1
